ffi/nodejs/swig.i

```diff
@@ -18,4 +18,17 @@
     $result = resolver->GetPromise();
 %}
 
+%inline %{
+    uint8_t *uint_array(int size) {
+        return (uint8_t *) malloc(size*sizeof(uint8_t));
```

Comment:
can you check that the `sizeof(uint8_t)` is strictly `1`? May be something wrong with the Swig and it uses different size.

Reply:
I don't get your question here, could you expand it a bit? being a `uint8_t` I assume it will always be 1 byte

Reply:
I am suggesting the reason for the bug. While `uint8_t` is clearly one byte as it says, it does not mean that the bug is in Swig and on some specific platform it does not maps to a one byte type

Reply:
but how could the bug be there if, checking the bytes logged by rust, I get exactly the same bytes that I get using `hd` on the consignment file?